src/lib/routes.js

```diff
@@ -26,23 +27,39 @@ module.exports = server => {
             next(new server.errors.InternalServerError(err));
           });
       }
-
-      next(new server.errors.BadRequestError(`Make sure the date format is '${dateFormat}' (time is optional)`));
+      next(new server.errors.BadRequestError(`Make sure the date format is "${dateFormat}" (time is optional)`));
     } else {
-      next(new server.errors.BadRequestError('Must provide a \'from\' date parameter, and optionally a \'to\' date'));
+      next(new server.errors.BadRequestError('Must provide a "from" date parameter, and optionally a "to" date'));
     }
   });
 
+  server.get('/dashboard', (req, res) => {
+    dashboardModel()
+      .then(props => res.render(LevDashboard, props))
+      .catch(err => server.log.error(err));
+  });
+
+  server.get('/dashboard/data', (req, res, next) => { // eslint-disable-line consistent-return
+    return dashboardModel()
+      .then(data => JSON.stringify(data, null, 2))
+      .then(res.end.bind(res))
+      .catch(err => {
+        server.log.error(err);
+        next(new server.errors.InternalServerError(err));
```

Comment:
I think the error handler should do this.

I _think_ we just want:
```js
next(err);
```

But I'm not certain.

Reply:
Refactored